contentcuration/contentcuration/static/js/edit_channel/import/views.js

```diff
@@ -1,15 +1,16 @@
-var Vue = require('vue');
+import Vue from 'vue';
```

Comment:
@jayoshih 

New `import` syntax from es6 - behaves a little differently, but with webpack allows for ["tree shaking" aka dead code removal](https://webpack.js.org/guides/tree-shaking/#src/components/Sidebar/Sidebar.jsx).

Main thing to take into account in this awkward transition period between `import` and `require` is that `import` assumes there was an `export` property `default`, and makes one if there wasn't one defined.